index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Youtube Replica</title>
+<link rel="stylesheet" href="index.css">
+</head>
+<body>
```

Comment:
please Insert a Meta tag so that the header can be compatible with different devices